pkg/omf/omf.fish

```diff
@@ -21,6 +21,10 @@ function omf::dim
   set_color $fish_color_autosuggestion ^/dev/null; or set_color 555
 end
 
+function omf::under
```

Comment:
This isn't being used anymore. I liked underline for section titles, better than `omf::dim`